Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -451,12 +448,12 @@ private Arc AlignmentArcToSpeckle(CivilDB.AlignmentSubEntityArc arc)
     Point2d midPoint = chordMid.Add(unitMidVector.MultiplyBy(sagitta));
     try
     {
-      if (arc.GreaterThan180) // sometimes this prop throws an exception??
+      if (arc.GreaterThan180) // this can throw
       {
         midPoint = chordMid.Add(unitMidVector.Negate().MultiplyBy(2 * arc.Radius - sagitta));
       }
     }
-    catch { }
+    catch (InvalidOperationException){ } // The property gets an invalid value according to the entity's constraint type.
```

Comment:
Not sur I follow the comment, are things in a usable/correct state if this exception has been caught?

Reply:
yes, clarified in the comments